docs/api_reference2.rst

```diff
@@ -0,0 +1,7 @@
+merge_sbom module
+=================
+
+.. automodule:: merge_sbom
```

Comment:
I think the "merge_sbom" text can be replaced with "surfactant" to generate an api reference for surfactant (rather than the merge_sbom utility script that doesn't get pip installed).